lib/lamby/debug.rb

```diff
@@ -6,7 +6,7 @@ module Debug
 
     def on?(event)
       params = event['multiValueQueryStringParameters'] || event['queryStringParameters']
-      (Rails.env.development? || ENV['LAMBY_DEBUG']) && params && params['debug'] == '1'
+      ENV['LAMBY_DEBUG'] && params && params['debug'] == '1'
```

Comment:
Would you mind making a private helper method called `development?` and use `RACK_ENV` and `RAILS_ENV` ENV checks in that helper method. I'd like to keep the safeguard in.

Reply:
Hi @metaskills - yup you're right! 

Looks like I missed the `present?` calls. I suspect tests are still passing because they're probably loading Rails (and therefore also ActiveSupport) for integration.

I don't see any references to `with_indifferent_access` or other ActiveSupport-iness though, so it should be pretty easy to resolve.

I'll take care of your suggestions later today, thanks!